### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by ShaunakJoshi1407**
-Added requirements of pf9ctl.
-Added usage along with description of the commands in use.
-Explained each command with a sample code snippet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,266 @@
...
+
+Global Flags:
+      --verbose   print verbose logs
+
+Use "pf9ctl config [command] --help" for more information about a command.
+```  
+
+- **Check Node**
+
+  This command will perform the prerequisite check before a node can be added to the cluster. It checks for the following:
```
Comment (anupbarve) [inline README.md:84]: s/check/checks
Could we add which checks are mandatory and which ones are optional ?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,266 @@
...
+✓ DiskCheck
+✓ MemoryCheck
+✓ PortCheck
+✓ Existing Kubernetes Cluster Check
+
+✓ Completed Pre-Requisite Checks successfully
+```
+- **prep-node**
+
+ This command prepares a node to be ready to be added to a cluster. It installs all the required pre-requisites on the nodes and installs the Platform9 host agent and other required software on the nodes.
```
Comment (anupbarve) [inline README.md:155]: Replace above text with:

This command onboards a node. It installs platform9 packages on the host. After completion of this command, the node is available to be managed on the Platform9 control plane.

**Review by anupbarve — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ShaunakJoshi1407) [reply README.md:84]: Ok will add that

File: README.md (hunk 2 above)
Comment (ShaunakJoshi1407) [reply README.md:155]: Ok will replace this text

**Review by anupbarve — APPROVED** [review]

**Pull request merged**